theia-rpm-build-docker/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "theia",
+  "version": "0.0.1",
+  "license": "EPL-2.0",
+  "engines" : {
+	  "node":"10.x"
+  },
+  "dependencies": {
+      "@theia/callhierarchy": "latest",
+      "@theia/file-search": "latest",
+      "@theia/git": "latest",
+      "@theia/json": "latest",
+      "@theia/markers": "latest",
+      "@theia/merge-conflicts": "latest",
+      "@theia/messages": "latest",
+      "@theia/mini-browser": "latest",
+      "@theia/navigator": "latest",
+      "@theia/outline-view": "latest",
+      "@theia/preferences": "latest",
+      "@theia/preview": "latest",
+      "@theia/search-in-workspace": "latest",
+      "@theia/terminal": "latest",
+      "@theia/textmate-grammars": "latest",
+      "typescript": "latest"
+  },
+  "devDependencies": {
+      "@theia/cli": "latest"
+  },
+  "scripts": {
+      "prebuild": "npm install -g yarn",
+      "prebuild-rpm": "ln -s \"$( pwd )\" ~/rpmbuild && npm install -g speculate",
+      "build-rpm": "speculate && rpmbuild -bb SPECS/theia.spec",
+      "start": "npm run prebuild && yarn theia start"
+  },
+  "spec": {
+    "rebuild": false,
+    "prune": false,
+    "nodeVersion": "> 10.0.0",
+    "requires": [
+      "nodejs",
+      "python"
```

Comment:
I noticed that `git` is currently missing, which causes exceptions at Theia startup. If we add it here, will it then be installed along with the Theia RPM? The version may be a bit old vs what Theia is expecting, however. 

Removing extension `@theia/git` would also fix the issue.

Reply:
BTW, general question: here CentOS 7 is used to generate and install/test the RPM. Would the RPM be installable on later versions of the OS as well?

Reply:
I removed `@theia/git` extension for now to keep things simple.

Regarding using a later version of the OS, according to the [speculate](https://github.com/bbc/speculate) docs, it currently supports CentOS 7. I made an [issue](https://github.com/bbc/speculate/issues/75) asking if they plan to support CentOS 8. The module is backed by BBC news so it probably has more support than a normal open source project.